games-simulation/openrct2/files/openrct2.initd

```diff
@@ -0,0 +1,32 @@
+#!/sbin/openrc-run
```

Comment:
Yes, I am familiar with `SVCNAME`. So I know, how that would like be implemented. Did such thing already with `games-server/minecraft-server`. Despite of this, I am currently a little bit skeptical. Multiplayer is still early and tends to desync. At least in my tests, I observed, running multiple servers parallel tended more to desync. At least for me here. This could be also just bad luck, as upstream warns anyway, that games could desync. I am unsure about that one. But I you think, it would be still a good idea, I could implement that.

Reply:
Should multiple instances of the same process be independent? Do you have a reason to believe that they affect each other?

Reply:
How do you define independent? From what I see, multiple processes would be in this case run with each different directory, where config etc. are stored. I wouldn't really belive, that this could be a couse of desync, but I am not an expert here. That what I meant, I am a little bit skeptical. We could just implement this and see how it goes for other users.

Reply:
Allright, I have now added multi instance support.
So you can use a single server with `/etc/init.d/openrct2` and add more with `/etc/init.d/openrct2.foo` etc. I used as default instance name `main`. Also did a small change, that the server scenario should be placed into the subfolder `scenario`, as it's also automatically created by the server.